index.html

```diff
@@ -259,11 +259,12 @@
           </p>
           <ul>
             <li>
-              <code>http://fedora.info/definitions/repository#EmbedResources</code>:
-              Requires a server to include representations of any contained resources in the response.
+              <code>http://www.w3.org/ns/oa#PreferContainedDescriptions</code>:
```

Comment:
@acoburn : that is a pretty reasonable argument. I will not have time to dig more deeply into the Web Annotations spec at the moment, but would be interested to hear if @azaroth42 sees the functional equivalence of the proposed scenario.